specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/examples/NetworkManagerDeploymentRulesGet.json

```diff
@@ -0,0 +1,40 @@
+{
+  "parameters": {
```

Comment:
Change to NetworkManagerCommittedSecurityConfigurationRulesGet
@keflem Commit will make sense with post so it depends on what we decide here.
Hold on the name change until we close.